frontend_app_admin_workspace_user/src/action.async.js

```diff
@@ -96,18 +96,28 @@ export const putUserProfile = (apiUrl, userId, newProfile) =>
     method: 'PUT'
   })
 
-export const postAddUser = (apiUrl, name, email, profile, emailNotif, password) =>
+export const postAddUser = (apiUrl, name, username, email, profile, emailNotif, password) =>
```

Comment:
I think you can rename `name` parameter to publicName to be consistent

Reply:
Changed